crates/jstz_cli/src/term/emoji.rs

```diff
@@ -0,0 +1,4 @@
+use console::Emoji;
+
+pub static WARN: Emoji = Emoji("▲", "");
```

Comment:
isn't there a better WARN emoji?

Reply:
Yes, but the benefit of this emoji is that it works well with ANSI colours